TYPO3.Neos/Documentation/HowTos/CustomLoginScreen.rst

```diff
@@ -34,7 +34,8 @@ inside a ``Views.yaml`` to include your custom logo partial::
       partialRootPathPattern: resource://Neos.Demo/Private/Partials
 
 
-With this example Neos would assume a logo in ``Packages/Sites/Neos.Demo/Resources/Private/Partials/Login/Logo.hml``
+With this example Neos would assume a logo in ``Packages/Sites/Neos.Demo/Resources/Private/Partials/Login/Logo.html``
+Note: Depending on your logo you might want to adjust the height for the css class ``.neos[style] .neos-background-image-clone``
```

Comment:
This change is obsolete, and the seperate layout element is gone: https://github.com/neos/neos-development-collection/pull/566/files#diff-2d322c760eb62cf1d3f4102279ec89baL24